contracts/pools/balancer/BMath.sol

```diff
@@ -339,9 +337,10 @@ contract BMath is BConst, BNum {
         // Do reverse order of fees charged in joinswap_ExternAmountIn, this way 
         //     ``` pAo == joinswap_ExternAmountIn(Ti, joinswap_PoolAmountOut(pAo, Ti)) ```
         //uint tAi = tAiAfterFee / (1 - (1-weightTi) * swapFee) ;
+        
         uint zar = bmul(bsub(BONE, normalizedWeight), _swapFee);
         tokenAmountIn = bdiv(tokenAmountInAfterFee, bsub(BONE, zar));
```

Comment:
These two lines are no longer being used in this method. So, remove them? But there may be more to it than this. The method seems to be changed a lot, is it still performing to its intent by callers?